usage/dist/pom.xml

```diff
@@ -63,6 +63,35 @@
     <build>
         <plugins>
             <plugin>
+                <artifactId>maven-dependency-plugin</artifactId>
+                <!-- copy the config file from the CLI project -->
+                <executions>
+                    <execution>
+                        <id>copy</id>
```

Comment:
I wondered if there was such a thing.  That's cool.  I'd have used it if I'd known.

However I do like having an artifact for the `bom` as it is easier to access directly.  Maybe it makes sense for that to be a standard pattern for bundles.  We'll see.  For now I'll leave it but add a comment about your suggestion.